### PR TITLE
Feature: Date created and status is added in table format.

### PR DESCRIPTION
**Pull request by insaurabh**
New method is added in `IssueModel.php` for `state` and `created_at`. To show these data table format is used.

New Feature Submissions:
 - [x] Have you lint your code locally prior to submission?

Changes to Core Features:
 - [x]  Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

**Comment by insaurabh** [comment]
Output

image

**Comment by icanhazstring** [comment]
Hi @insaurabh thank you 👍 
Could you please update the console output with the current master?
I have merged and solved #10 #9 which changes the output format quite a bit.

**Comment by icanhazstring** [comment]
Thanks for updating. And Merged 👍

**Comment by insaurabh** [comment]
@icanhazstring 

Thank you for accepting the pull request. 

If you are working on any open source project on which I can contribute and learn at some time. Please let me know.

**Comment by icanhazstring** [comment]
You are welcome. I think https://github.com/icanhazstring/systemctl-php/issues could be some place where you can contribute :)

File: src/Command/RandomIssueCommand.php [hunk 1]
```diff
@@ -86,13 +86,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $randomIssue = $searchIssueModel->getItems()[$randomIssueIndex];
 
+        $io->section($randomIssue->getTitle());
+
+        $io->newLine(1);
+
```
Comment (DenverCoder1) [inline src/Command/RandomIssueCommand.php:92]: Is there a reason to keep this? The title is already in the table so I don't think it needs to appear twice.

Alternatively, if it looks better, we could remove the title from the table and have the title as a heading instead.

Comment (icanhazstring) [reply src/Command/RandomIssueCommand.php:92]: You are correct, I didn't see that. The additional section here is not needed. If you want. Create a new PR removing it ;)

**Pull request merged**